pages/docs/couple-your-code/couple-your-code-mesh-and-data-access.md

```diff
@@ -87,6 +140,44 @@ precice.finalize(); // frees data structures and closes communication channels
 turnOffSolver();
 ```
 
+</div>
+<div role="tabpanel" class="tab-pane" id="python-4" markdown="1">
+
+```python
+turn_on_solver() # e.g. setup and partition mesh
+precice = precice.Participant("FluidSolver", "precice-config.xml", rank, size) # Initialize participant
+
+mesh_dim = precice.get_mesh_dimensions("FluidMesh")
+
+# Define coords of dimensions (vertex_size*mesh_dim)
```

Comment:
Let's really try to follow the C++ code here 1:1: intialization with zero and then a comment for the actual data. Be careful, `coords` needs to be multi-dimensional.